tests/parsers/aws_elb_access.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+"""Tests for Apache access log parser."""
+
+import unittest
+
+from plaso.parsers import aws_elb_access
+from tests.parsers import test_lib
+
+
+class AWSELBUnitTest(test_lib.ParserTestCase):
+  """Tests for AWS ELB access log parser."""
+
+  def testParse(self):
+    """Tests the Parse function."""
+    parser = aws_elb_access.AWSELBParser()
+    storage_writer = self._ParseFile(['aws_elb_access.log'], parser)
+
+    # Test number of events and wrarnings
+    self.assertEqual(storage_writer.number_of_events, 10)
+    self.assertEqual(storage_writer.number_of_extraction_warnings, 0)
+    self.assertEqual(storage_writer.number_of_recovery_warnings, 0)
+
+    # The order in which parser generates events is nondeterministic hence
+    # we sort the events.
+    events = list(storage_writer.GetSortedEvents())
+
+    expected_timestamps = ['2020-01-11 16:55:20.000000']
+    expected_event_values = {
+      "actions_executed": "waf,forward",
```

Comment:
style nit: continuation indentation should be 4 spaces

Reply:
Fixed.